plugins/module_utils/version.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""Provide version object to compare version numbers."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.module_utils.six import raise_from
+
+try:
+    from ansible.module_utils.compat.version import LooseVersion
+except ImportError:
+    try:
+        from distutils.version import LooseVersion
+    except ImportError as exc:
+        msg = 'To use this plugin or module with ansible-core < 2.11, you need to use Python < 3.12 with distutils.version present'
```

Comment:
```suggestion
        msg = ('To use this plugin or module with ansible-core < 2.11, '
               'you need to use Python < 3.12 with distutils.version present')
```
feel free to just Resolve

Reply:
I would add at least the period to the end

Reply:
I would keep the message in one line. About the period: regular `ImportError`s generated by Python also do not have one, so I think it's better to not add one here.